From 90fb8133e30c0cb6ecc5d56d47798a59846e7fab Mon Sep 17 00:00:00 2001
From: Andrew Grieve <agrieve@chromium.org>
Date: Tue, 14 Jun 2022 15:10:47 +0000
Subject: [PATCH 06/12] Android: Allow java_library targets to set
 mergeable_android_manifests

Rather than force these to be a part of android_resources or java_group.

Bug: None
Change-Id: Ie657d0849dc0dd04b16acde1bf3bdf9a35ef7796
Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/3704197
Auto-Submit: Andrew Grieve <agrieve@chromium.org>
Commit-Queue: Andrew Grieve <agrieve@chromium.org>
Reviewed-by: Sam Maier <smaier@chromium.org>
Commit-Queue: Sam Maier <smaier@chromium.org>
Cr-Commit-Position: refs/heads/main@{#1013960}
---
 build/config/android/internal_rules.gni | 1 +
 1 file changed, 1 insertion(+)

diff --git a/build/config/android/internal_rules.gni b/build/config/android/internal_rules.gni
index b5eaa27b4fb57..f05a02c5379e8 100644
--- a/build/config/android/internal_rules.gni
+++ b/build/config/android/internal_rules.gni
@@ -3531,6 +3531,7 @@ if (enable_java_templates) {
                                "input_jars_paths",
                                "low_classpath_priority",
                                "main_class",
+                               "mergeable_android_manifests",
                                "proguard_configs",
                                "proguard_enabled",
                                "proguard_mapping_path",
-- 
2.37.2

